.ansible-lint

```diff
@@ -1,2 +1,5 @@
 exclude_paths:
 - .travis.yml
+
+warn_list:
```

Comment:
I would move this to the failing tasks instead so this will only match existing warnings and not any future changes to the role.